src/ext/span.c

```diff
@@ -21,8 +21,12 @@ static void _free_span(ddtrace_span_t *span) {
     ddtrace_zval_ptr_dtor(span->span_data);
     efree(span->span_data);
     if (span->exception) {
+#if PHP_VERSION_ID < 70000
         ddtrace_zval_ptr_dtor(span->exception);
```

Comment:
On PHP 5, `ddtrace_zval_ptr_dtor` evaluates to `zval_dtor`, which will not free the zval itself, so theoretically this would leak. I need to switch t his to `zval_ptr_dtor` and then it should be good.